src/main/java/Handler/EchoHandler.java

```diff
@@ -12,15 +13,23 @@ public Response generateResponse(Request request) {
     String messageBody = "";
     switch (method) {
       case "GET": 
-        return new Response.Builder(HttpStatusCode.OK)
-                           .messageBody(createMessageBody())
-                           .build();
+        return buildGETResponse();
       default: 
         return new Response.Builder(HttpStatusCode.METHOD_NOT_ALLOWED)
                            .build();
     }
   }
 
+  private Response buildGETResponse() {
```

Comment:
I noticed that in FileHandler.java you have a method called `buildGetResponse`. Stick with that capitalization style to be consistent